lua/entities/gmod_wire_expression2/core/custom/cfc_number.lua

```diff
@@ -0,0 +1,123 @@
+E2Lib.RegisterExtension( "cfc_e2_lib", true )
+
+--https://en.wikipedia.org/wiki/List_of_trigonometric_identities
+
+local math   = math
+local sin    = math.sin
+local cos    = math.cos
+local acos   = math.acos
+local asin   = math.asin
+local atan   = math.atan
+local sqrt   = math.sqrt
+
+--https://en.wikipedia.org/wiki/Versine
+e2function number ver(number num)
+    return 1 - cos(num)
+end
+
+--https://en.wikipedia.org/wiki/Versine
+e2function number vcs(number num)
```

Comment:
```suggestion
e2function number coversin(number num)
```

Reply:
I didn't bother changing all of them but all of the functions need to be named better.